usaspending_api/api_contracts/contracts/v2/reporting/agencies/toptier_code/fiscal_year/fiscal_period/unlinked_awards.md

```diff
@@ -0,0 +1,37 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Agencies' Unlinked Awards [/api/v2/reporting/agencies/{toptier_code}/{fiscal_year}/{fiscal_period}/unlinked_awards/{type}/]
```

Comment:
Just noticed that since `{type}` is no long a parameter so the API contract would be named `type.md`  to follow the naming convention like the other values in the URL path. The python view file can remain as-is

`usaspending_api/api_contracts/contracts/v2/reporting/agencies/toptier_code/fiscal_year/fiscal_period/unlinked_awards/type.md`

Reply:
Fixed this